hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/BackupSystemTable.java

```diff
@@ -1736,11 +1739,13 @@ static Scan createScanForOrigBulkLoadedFiles(TableName table) {
     return scan;
   }
 
+  @SuppressWarnings("StringSplitter")
```

Comment:
Since we have shaded guava, I think it is OK for use to make use of guava's Splitter.

Reply:
There's nothing wrong with string.split in these applications and when I used Splitter it caused more trouble than it was worth.